src/util.ts

```diff
@@ -138,7 +156,7 @@ export async function getFriends(pid: number): Promise<GetUserFriendPIDsResponse
 	}
 }
 
-export async function getCDNFileStream(key: string): Promise<Readable | fs.ReadStream | null> {
+export async function getCDNFileStream(key: string): Promise<Readable | null> {
```

Comment:
I'm a little concerned about the removal of this type, why was this removed? `getCDNFileStream` does not need to pull from s3, if disabled it will pull a file from disk using `getLocalCDNFile` which returns `fs.ReadStream`?

Reply:
Readable is the generic stream class, and is what fs.ReadStream inherits from

This just simplifies the type

Reply:
Sounds good to me :+1: